### ISSUE TITLE
Spectral Rift (temp) Point to open a read time (BB #505)

### ISSUE THREAD

**Issue by OregonAsari** [OregonCore/OregonCore#505]
This issue was migrated from bitbucket.
**Original Reporter:** danqu
**Original Date:** 26.02.2014 12:25:26 GMT+0000
**Original Priority:** major
**Original Type:** bug
**Original State:** resolved
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/505

<hr>

gameobject，Spectral Rift (temp) Point to open a read time

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** LordUsagi
**Original Date:** 26.02.2014 17:32:50 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/505/#comment-8645428

<hr>

```
#!c++

# HG changeset patch
# User LordUsagi
# Date 1393435914 14400
#      Wed Feb 26 13:31:54 2014 -0400
# Node ID 8104980a0fdf84162c822b238e679fd2883421cb
# Parent  ad20e65a46d9f3a9061fbca3fd22f22c3e190648
[Scripts/Objects] Fix Spectral Rift cast time. Closes #505.

diff -r ad20e65a46d9 -r 8104980a0fdf src/game/ScriptedCreature.cpp
--- a/src/game/ScriptedCreature.cpp Wed Feb 26 12:58:40 2014 -0400
+++ b/src/game/ScriptedCreature.cpp Wed Feb 26 13:31:54 2014 -0400
@@ -660,7 +660,7 @@
     goInfo = GOBJECT(187055);
     if (goInfo)
         if (goInfo->type == GAMEOBJECT_TYPE_GOOBER)
-            goInfo->goober.lockId = 57; // need LOCKTYPE_QUICK_OPEN
+            goInfo->type = GAMEOBJECT_TYPE_SPELLCASTER;

     // Naxxramas : Sapphiron Birth
     goInfo = GOBJECT(181356);

```

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** Zaffy
**Original Date:** 26.02.2014 18:02:28 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/505/#comment-8645990

<hr>

[Scripts/Objects] Fix Spectral Rift cast time. Closes #505.

→ <<cset 839cc134ab2b>>
